R/lms.R

```diff
@@ -33,8 +33,7 @@
 #' @export
 
 lms <- function(data, formula, grp_id, obs_id = NULL, trans_out = TRUE) {
-  require(tidyverse)
-  require(dtplyr)
+  library(data.table)
```

Comment:
We do not allow any require or library calls within our package, as these affect the searchpath!
See https://r-pkgs.org/code.html#sec-code-r-landscape